models/glacier/avalanche_c/glacier_avalanche_c_schema.yml

```diff
@@ -1,16 +1,16 @@
 version: 2
 
 models:
-  - name: glacier_avalanche_c_trades
+  - name: glacier_v3_avalanche_c_trades
```

Comment:
You still need the model definition for `glacier_avalanche_c_trades` in this file.